### PR TITLE
Move ng-service enablement to .ng-dev github config, enable it for this repo. 

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: ng-dev/utils/ng-dev-service.ts [hunk 1]
```diff
@@ -30,49 +32,56 @@ const firebaseConfig = {
...
 
-        Log.error('  ✘  You must be logged in to run this command\n');
-        Log.log('Log in by running the following command:');
-        Log.log('  yarn ng-dev auth login');
-        argv.exit(1, new Error('The user is not logged in'));
-      })
+          Log.error('  ✘  You must be logged in to run this command\n');
+          Log.log('Log in by running the following command:');
+          Log.log('  yarn ng-dev auth login');
+          argv.exit(1, new Error('The user is not logged in'));
```
Comment (devversion) [inline ng-dev/utils/ng-dev-service.ts:83]: I don't think we use `argv.exit` anywhere else. Can we just throw the error like elsewhere and Yargs will pick it up?

**Review by devversion — APPROVED** [review]

**Review by devversion — APPROVED** [review]

File: ng-dev/utils/ng-dev-service.ts (hunk 1 above)
Comment (josephperrott) [reply ng-dev/utils/ng-dev-service.ts]: Done.  I will come back to how we handle exiting as we currently are very inconsistent in a follow up.

File: .ng-dev/github.mts [hunk 2]
```diff
@@ -5,4 +5,5 @@ export const github: GithubConfig = {
   owner: 'angular',
   name: 'dev-infra',
   mainBranchName: 'main',
+  useNgDevService: true,
```
Comment (devversion) [inline .ng-dev/github.mts:8]: I wonder if we should name this

```suggestion
  useNgDevAuthService: true,
```

Comment (devversion) [reply .ng-dev/github.mts:8]: nothing blocking, but just as an idea

**Pull request closed without merging**